src/main/resources/application.yaml

```diff
@@ -65,4 +66,7 @@ feign:
     headers:
       - authorization
       - serviceauthorization
-documentStoreUrl: ${Document_STORE_URL:http://localhost:3453/documents}
+      - user-roles
+      - user-id
+documentStoreUrl: ${Document_STORE_URL:http://localhost:4506/documents}
+caseDataStoreUrl: ${CASE_DATA_STORE_URL:http://localhost:3453/data}
```

Comment:
Need to add the URLs for all the higher environments as well

Reply:
it will come under the tf files for higher env. I am not sure but need to ask with Nitish. Perhaps, He might updated.